plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -166,20 +174,35 @@ func PrioritizeNodes(pod *api.Pod, podLister algorithm.PodLister, priorityConfig
 		priorityFunc := priorityConfig.Function
 		prioritizedList, err := priorityFunc(pod, podLister, minionLister)
 		if err != nil {
-			return algorithm.HostPriorityList{}, err
+			return schedulerapi.HostPriorityList{}, err
 		}
 		for _, hostEntry := range prioritizedList {
 			combinedScores[hostEntry.Host] += hostEntry.Score * weight
 		}
 	}
+	if extender != nil && minionLister != nil {
+		nodes, err := minionLister.List()
+		if err != nil {
+			return schedulerapi.HostPriorityList{}, err
+		}
+
+		prioritizedList, err := extender.Prioritize(pod, &nodes)
```

Comment:
Is there some reason why you made this interface different from priorityFunc() ?
In particular, priorityFunc() has access to all pods (whereas extender.Prioritize() takes just the pod you are trying to schedule), and only the set of nodes that passed the predicate checks (whereas extender.Prioritize() takes the set of all nodes).